README.md

```diff
@@ -495,6 +495,30 @@ The following are some endpoints that accept Chargebee tokens for processing sub
 
 Please refer to the [Chargebee API Docs](https://apidocs.chargebee.com/docs/api) for subsequent integration steps.
 
+### Upgrade or Downgrade Subscription
+
+When a user changes their subscription level from a lower price plan to a higher price plan, it's considered an upgrade. On the other hand, when a user switches from a higher-price plan to a lower-price plan, it's considered a downgrade.
+
+In the case of the Apple App Store, you can arrange the subscriptions using the drag-and-drop option in **Edit Subscription Order** in App Store Connect. [Learn more](https://developer.apple.com/app-store/subscriptions/#ranking).
+
+#### Invoke Upgrade or Downgrade Subscription Flow in your App
+
+The `showManageSubscriptionsSettings()` function is designed to invoke the upgrade/downgrade flow in your app using Chargebee's iOS SDKs. The function determines the appropriate method to use based on the iOS version available on the device.
+
+If the device is running iOS 15.0 or later, the function calls `UIApplication.showManageSubscriptions()` to display the native subscription management interface provided by the operating system. For devices running iOS versions prior to 15.0, the function calls `UIApplication.showExternalManageSubscriptions()`, which opens the Apple App Store subscriptions settings page.
+
+```swift
+public func showManageSubscriptionsSettings() {
+        if #available(iOS 15.0, *) {
+            UIApplication.showManageSubscriptions()
+        } else {
+            UIApplication.showExternalManageSubscriptions()
+        }
+    }
```

Comment:
This should be the code snippet of what the app needs to do and not the implementation itself. 
Can you fix this @cb-gadagoju ?

Reply:
@cb-gadagoju and @cb-haripriyan I have made the changes.